code-challenges.test.js

```diff
@@ -18,36 +20,70 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+var colors1 = ["purple", "yellow", "blue", "pink" , "green"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+var colors2 = ["chartreuse", "saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// define your test method
+describe("shuffle", () => {
+    // give it the parameters to test
+    test ("returns the shifted array ", () =>{
+        // give expected output
+        expect(shuffle(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+        expect(shuffle(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+    })
+})
+
+var colors1 = ["purple", "yellow", "blue", "pink" , "green"]
+var colors2 = ["chartreuse", "saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// b) Create the function that makes the test pass.
+// make function
+const shuffle = (array) => {
```

Comment:
I can see this one didn't get finished... and theres no notes as to how the logic would get worked out. Shoot me a message on slack or respond to this comment with the logic that you would have followed.